templates/20-04/aws.html

```diff
@@ -0,0 +1,232 @@
+      </div>
+      <div class="col">
+        <p>
+          Transitioning to an Ubuntu version with active standard support (Ubuntu 22.04 LTS or 24.04 LTS) is crucial for performance, hardware compatibility, and access to new technologies, making it ideal for new workloads. However, for existing deployments, the process may be more complex.
+        </p>
+        <div class="p-cta-block">
+          <a href="https://ubuntu.com/about/release-cycle">Learn more about the Ubuntu lifecycle and release&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <hr class="p-rule" />
+      <div class="p-section--shallow">
+        <h2>What are your options?</h2>
+      </div>
+
+      <ol class="p-stepped-list--detailed">
+        <li class="p-stepped-list__item">
+          <h3 class="p-stepped-list__title">
+            Upgrade to an Ubuntu version with active standard support (Ubuntu 22.04&nbsp;LTS or 24.04&nbsp;LTS)
+          </h3>
+          <div class="p-stepped-list__content">
+            <h4 class="p-heading--5">For Ubuntu 22.04 LTS (coverage through 2027):</h4>
+            <hr class="p-rule--muted u-no-margin--bottom" />
+            <ul class="p-list--divided">
+              <li class="p-list__item has-bullet"><a href="/server/docs/upgrade-introduction">Upgrade directly from Ubuntu 20.04 LTS</a></li>
+              <li class="p-list__item has-bullet">
+                <a href="https://aws.amazon.com/marketplace/pp/prodview-dxdx74ozlxsl2">Launch a new instance of Ubuntu 22.04 LTS</a>
+              </li>
+            </ul>
+            <h4 class="p-heading--5">For Ubuntu 24.04 LTS (coverage through 2029):</h4>
+            <hr class="p-rule--muted u-no-margin--bottom" />
+            <ul class="p-list--divided">
+              <li class="p-list__item has-bullet"><a href="/server/docs/upgrade-introduction">Upgrade to Ubuntu 22.04 LTS first and then upgrade to 24.04 LTS</a></li>
+              <li class="p-list__item has-bullet">
+                <a href="https://aws.amazon.com/marketplace/pp/prodview-kbkyvwyqubv2g">Launch a new instance of Ubuntu 24.04 LTS</a>
+              </li>
+              <li class="p-list__item has-bullet">
+                <a href="#get-in-touch">Contact us to purchase annual subscription tokens</a>
+              </li>
+            </ul>
+          </div>
+        </li>
+
+        <li class="p-stepped-list__item">
+          <h3 class="p-stepped-list__title">
+            Attach an Ubuntu Pro 20.04 subscription to expand your security maintenance (support through 2030).
+          </h3>
+          <div class="p-stepped-list__content">
+            <ul class="p-list--divided">
+              <li class="p-list__item has-bullet">
+                <a href="/tutorials/how-to-upgrade-ubuntu-lts-to-ubuntu-pro-on-aws-using-aws-license-manager#1-overview">In-place upgrade from Ubuntu 20.04 to Ubuntu Pro 20.04 with no downtime</a>
```

Comment:
```suggestion
                <a href="/tutorials/how-to-upgrade-ubuntu-lts-to-ubuntu-pro-on-aws-using-aws-license-manager#1-overview">In-place upgrade from Ubuntu 20.04 to Ubuntu Pro 20.04</a>
```
"with no downtime" isn't there on the copydoc